examples/configs/src/view-configs/eng.js

```diff
@@ -139,7 +139,7 @@ export const eng2019 = {
     {
       component: 'obsSetSizes',
       x: 5,
-      y: 4,
+      y: 0,
       w: 4,
       h: 4,
     },
```

Comment:
TODO: remove changes to this file. Changes were done only for convenience when developing